packages/eslint-plugin/src/rules/no-unnecessary-parameter-property-assignment.ts

```diff
@@ -0,0 +1,233 @@
+import { DefinitionType } from '@typescript-eslint/scope-manager';
+import type { TSESTree } from '@typescript-eslint/utils';
+import { AST_NODE_TYPES, ASTUtils } from '@typescript-eslint/utils';
+
+import { createRule, getStaticStringValue, nullThrows } from '../util';
+
+const UNNECESSARY_OPERATORS = new Set(['=', '&&=', '||=', '??=']);
+
+export default createRule({
+  name: 'no-unnecessary-parameter-property-assignment',
+  meta: {
+    docs: {
+      description:
+        'Disallow unnecessary assignment of constructor property parameter',
+    },
+    fixable: 'code',
```

Comment:
Is this true? I'm not able to get it to fix my code, and I don't see where a fixer is passed to `context.report`.

Reply:
Aha! You're right - this was an oversight. https://github.com/eslint/eslint/issues/18008 would have caught it. 

Are you up for filing an issue and/or sending a PR @dasa? It'd be a great contribution  😁